src/main/java/uk/gov/ons/census/action/model/entity/FulfilmentsToSend.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.ons.census.action.model.entity;
+
+import com.vladmihalcea.hibernate.type.json.JsonBinaryType;
+import java.util.UUID;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import lombok.Data;
+import org.hibernate.annotations.Type;
+import org.hibernate.annotations.TypeDef;
+import org.hibernate.annotations.TypeDefs;
+
+@Entity
+@TypeDefs({@TypeDef(name = "jsonb", typeClass = JsonBinaryType.class)})
+@Data
+public class FulfilmentsToSend {
+
+  @Id
+  @Column(columnDefinition = "serial")
+  private long id;
+
+  @Column private String fulfilmentCode;
+
+  @Type(type = "jsonb")
+  @Column(columnDefinition = "jsonb", nullable = false)
+  private String messageData;
+
+  private Integer quantity;
+
+  private UUID batchId;
```

Comment:
The definition should match Action Scheduler 100% - i.e. this column should be declared nullable

Reply:
also, why isn't this declared `@Column`?